app/jobs/reports/monthly_account_reuse_report.rb

```diff
@@ -0,0 +1,143 @@
+          , agencies_per_user.num_agencies
+          FROM (
+              SELECT
+                  COUNT(DISTINCT agencies.id) AS num_agencies
+              , identities.user_id
+              FROM 
+                  identities
+              JOIN 
+                  service_providers sp ON identities.service_provider = sp.issuer
+              JOIN 
+                  agencies ON sp.agency_id = agencies.id
+              WHERE
+                  identities.last_ial2_authenticated_at IS NOT NULL
+              AND
+                  identities.verified_at < %{query_date}
+              GROUP BY 
+                  identities.user_id
+          ) agencies_per_user
+          GROUP BY 
+              agencies_per_user.num_agencies
+          HAVING agencies_per_user.num_agencies > 1
+          ORDER BY 
+              num_agencies ASC
+      SQL
+
+      proofed_sql = format(<<-SQL, params)
+          SELECT
+            COUNT(*) AS num_proofed
+          FROM
+            profiles
+          WHERE
+            profiles.active = TRUE
+          AND
+            profiles.activated_at < %{query_date}
+      SQL
+
+      agency_results = transaction_with_timeout do
+        ActiveRecord::Base.connection.execute(agency_sql)
+      end
+
+      proofed_results = transaction_with_timeout do
+        ActiveRecord::Base.connection.execute(proofed_sql)
+      end
+
+      reuse_report = agency_results.as_json
+
+      total_proofed = proofed_results.first['num_proofed']
+
+      total_reuse_stats = {
+        label: 'Total (all >1)',
+        num_users: 0,
+        percentage: 0,
+      }
+
+      if !reuse_report.empty?
+        reuse_report.each do |result_entry|
+          total_reuse_stats[:num_users] = total_reuse_stats[:num_users] + result_entry['num_users']
+        end
+
+        if total_proofed > 0
+          reuse_report.each_with_index do |result_entry, index|
+            reuse_report[index]['percentage'] =
+              result_entry['num_users'] / total_proofed.to_f * 100
+
+            total_reuse_stats[:percentage] += reuse_report[index]['percentage']
+          end
+        end
+      end
+
+      report_csv = []
+      report_csv << ["IDV app reuse rate #{stats_month}"]
+      report_csv << [
+        'Num. SPs',
+        'Num. users',
+        'Percentage',
+      ]
+
+      reuse_report.each do |result_entry|
+        report_csv << [
+          result_entry['num_agencies'],
+          result_entry['num_users'],
+          result_entry['percentage'],
+        ]
+      end
+      report_csv << [total_reuse_stats[:label], total_reuse_stats[:num_users],
+                     total_reuse_stats[:percentage]]
+
+      report_csv << []
+      report_csv << ['Total proofed identities']
+      report_csv << ["Total proofed identities (#{stats_month})", total_proofed]
+
+      {
+        report_date: first_day_of_report_month,
+        month: stats_month,
+        results: [report_csv],
+      }
+    end
+  end
+  end
```

Comment:
either one of these is extra or there's an alignment issue

Reply:
I'm surprised the lintfix doesn't catch/fix that?